src/Codeception/Command/Bootstrap.php

```diff
@@ -140,16 +140,6 @@ public function createGlobalConfig()
             ],
             'extensions' => [
                 'enabled' => ['Codeception\Extension\RunFailed']
-            ],
```

Comment:
not too related for this PR, but better now than never to remove this default config part
it has no use for most setups